deps/media-playback/media-playback/media.c

```diff
@@ -601,6 +602,38 @@ static int interrupt_callback(void *data)
 	return stop;
 }
 
+static void set_ffmpeg_options(char **opt_tokens, AVDictionary *opts)
+{
+	const char *equals = "=";
+	struct dstr key = {0};
+	struct dstr value = {0};
+	char **tokens = opt_tokens;
+	while (*tokens) {
+		char *token = *tokens;
+		size_t equals_pos = strcspn(token, equals);
+
+		if (equals_pos < strlen(token)) {
+			dstr_ncopy(&key, token, equals_pos);
+			dstr_copy(&value, token);
+			dstr_right(&value, &value, equals_pos + 1);
+
+			if (!dstr_is_empty(&key) &&
+				!dstr_is_empty(&value)) {
+
+				char *k = dstr_to_mbs(&key);
+				char *v = dstr_to_mbs(&value);
+				blog(LOG_INFO,
+					"Setting FFmpeg directive %s to %s",
```

Comment:
Use "option" instead of "directive".  This should allow clang-format to pass with:
```
				blog(LOG_INFO, "Setting FFmpeg option %s to %s",
					k, v);
```